documentation/volume-tracking/docs/index.md

```diff
@@ -0,0 +1,89 @@
+
+    1. Creating a library
+    2. Creating a pool
+
+    When we are using a library or a pool in a run, we are recording the ^^used volume^^ of each library or pool in a run, as well as the libraries that are used in a pool within a run.
+
+
+![process](img/pm-1.png)
+
+As mentioned above, the entire volume tracking process depends on the following concept:
+
+$$
+V_{required} < V_{original} - V_{used}
+$$
+
+The volume checking process involves primary aliquots that record the original volume of a library/pool or a run, and the summation of all the derived aliquots that are required. If the required volume is higher than the available volume, the volume check fails.
+
+## Libraries
+
+Assume that a library is created with a volume and concentration. The system then records this event by persisting a record in `aliquot` table with `aliquot_type` equal to `primary` and `source_type` equal to `library` along with the volume and concentration.
+
+The process diagram for the above use-case is depicted below.
+![process](img/pm-2.png)
+
+## Pools
+
+When creating a pool, a primary aliquot is created with `source_type` equal to `PacBio::Pool`. A Pool can be created using multiple libraries as well multiple requests. Derived aliquots are created for each library or request used in a pool. But we are only sending aliquots created by the library (we are not sending aliquots created by the request) to the warehouse. 
+
+<figure markdown="span">
+  ![process](img/pool-from-library.png)
+  <figcaption>Pool creation with samples</figcaption>
+</figure>
+
+=== "Used volume of the pool"
+
+    !!! note
+    
+        We are displaying the used volume of the pool as a badge (as shown in the figure below) and if the user enters a value less than the already-used volume an error prompt displayed in the page and the user would not be allowed to create/update the pool.
+
+    <figure markdown="span">
+      ![Pool Volume Check](img/vc-pool.png)
+      <figcaption>Pool creation with samples</figcaption>
+    </figure>
+
+=== "Available volume of libraries in the pool"
+
+    !!! note
+
+        We are displaying the available volume of each library, if the user enters a value higher than the available volume there would an error prompt displayed in the page and the user would not be allowed to create/update the pool.
+
+    <figure markdown="span">
+      ![Pooled Samples Volume Check](img/vc-pooled-samples.png)
+      <figcaption>Pool creation with samples</figcaption>
+    </figure>
+
+The process diagram for the above use-case is depicted below.
+![process](img/pm-2.png)
+
+## Run
+
+When creating a run, a primary aliquot is created with `source_type` as `Pacbio::Well` and a well can be created using multiple libraries as well as using multiple pools. Derived aliquots are created for each library or pool used in a run. We are publishing the following aliquots into the warehouse as part of run creation process:
```

Comment:
I don't know if we do create a primary aliquot for each well. Worth double checking.

Reply:
I did try it in UAT and looks like it does. Not sure whether if it's a bug though.

Reply:
I think we should only be publishing the used_aliquots in run well which are from
- a pool
- libraries within a pool
-  libraries that used directly within the run well